policy/modules/system/init.te

```diff
@@ -56,6 +56,12 @@ domain_entry_file(init_t, init_exec_t)
 kernel_domtrans_to(init_t, init_exec_t)
 role system_r types init_t;
 
+#
+# init_tmp_t is the type for /tmp/systemd-private-xxxx and /tmp/namespace-dev-xxxx
+#
+type init_tmp_t;
```

Comment:
I agree with Dominick, I'm not convinced this is a good change.

Reply:
OK. I will remove this type and grant the necessary relabel rules instead.